.github/workflows/python-poetry-release.yaml

```
@@ -79,20 +80,32 @@ jobs:
           poetry-version: ${{ inputs.poetry-version }}
           working-directory: ${{ inputs.working-directory }}
 
-      - name: Release to PyPI
-        uses: bakdata/ci-templates/actions/python-poetry-release@v1.18.2
+      - name: Create changelog
+        id: build_changelog
```

```suggestion
        id: build-changelog
```
use dash case; same as for `bump-version` step